cmd_clean/main.go

```diff
@@ -0,0 +1,10 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
```

Comment:
how about put cleanup_main.go into cmd directory

Reply:
how about this structure, since only one main.go in one directory.
repo
  +----cmd
  |     +----- cmd1
  |     |        +------ main.go
  |     |
  |     +----- cmd2
  |              +------ main.go
  |
  +----others.go
  +----libs.go
  +---- ...

Reply:
Yes, this looks good to me, similar with what Antrea does https://github.com/antrea-io/antrea/tree/main/cmd